docs/graphql/manual/guides/postgres/index.rst

```diff
@@ -0,0 +1,67 @@
+.. meta::
+   :description: Manage remote schemas with Hasura
+   :keywords: hasura, docs, remote schema
+
+.. _postgres:
+
+Postgres Guide
+==============
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+This page introduces Postgres including benefits and caveats, and it links to sections relevant to extending your GraphQL API by using Postgres features.
+
+About Postgres
+--------------
+
+`Postgres (or PostgreSQL) <https://www.postgresql.org/>`__ is a general-purpose object-relational database management system that uses and extends the SQL language.
+Postgres is free and open source and has been developed continuously over the last 30 years. Postgres strives to be reliable, robust and performant.
+
+Postgres supports advanced data types and advanced performance optimization, features that are otherwise only available in expensive commercial database systems.
+
+Benefits of Postgres
+--------------------
+
+The following are benefits of Postgres compared to other database management systems:
+
+- Postgres can be maintained easily because of its stability. The total cost of ownership is therefore low.
+- Postgres is designed to be extensible i.e. you can add custom functions using different programming languages.
+- You can define your own data types, index types etc.
+- If you need any support, an `active community <https://www.postgresql.org/community/>`__ is available to help. 
+
+Caveats
+-------
+
+Not all open source applications support Postgres as MySQL is generally a bit more widely spread. Also, when it comes to performance, MySQL has a slight edge.
```

Comment:
This cannot be used in docs without proper reference/context. I suggest we remove the `Caveats` section.